File: src/main/kotlin/com/tollwood/jpa/Company.kt
package com.tollwood.jpa

import javax.persistence.*

@Entity(name = "COMPANY")
data class Company(
        @Id
        @GeneratedValue(strategy = GenerationType.AUTO)
        val id: Long,

        var billNo: Long = 0,
        val name: String,
        val logo: String,
        @Embedded
        val address: Address,
        val phone: String,
        val email: String,
        val realEstateSupport: Boolean,
        val employeeSupport: Boolean,
        val billingSupport: Boolean,
        val customerSupport: Boolean
)

File: client/src/contractors/Company.ts
import Link from "../common/Links";
import { Address } from "../common/Address";
import BankDetails from "./BankDetails";

export default class Company {
  name: string = "";
  address: Address = new Address();
  billNo: number = 0;
  logo: string = "";
  phone: string = "";
  email: string = "";
  bankDetails: BankDetails = new BankDetails();
}
